code/modules/clothing/gloves/color.dm

```diff
@@ -80,6 +80,34 @@
 	. = ..()
 	siemens_coefficient = pick(0,0,0,0.5,0.5,0.5,0.75)
 
+/obj/item/clothing/gloves/cut
+	desc = "These gloves would protect the wearer from electric shock.. if the fingers were covered."
+	name = "fingerless insulated gloves"
+	icon_state = "yellowcut"
+	inhand_icon_state = "ygloves"
+	transfer_prints = TRUE
+
+/obj/item/clothing/gloves/cut/heirloom
+	desc = "The old gloves your great grandfather stole from Engineering, many moons ago. They've seen some tough times recently."
+
+/obj/item/clothing/gloves/color/yellow/attackby(obj/item/I, mob/user, params)
+	if(I.tool_behaviour == TOOL_WIRECUTTER)
+		if(can_be_cut && icon_state == initial(icon_state))//only if not dyed
+			to_chat(user, "<span class='notice'>You snip the fingertips off of [src].</span>")
+			I.play_tool_sound(src)
+			new /obj/item/clothing/gloves/cut(drop_location())
+			qdel(src)
+	..()
+
+/obj/item/clothing/gloves/color/fyellow/attackby(obj/item/I, mob/user, params)
+	if(I.tool_behaviour == TOOL_WIRECUTTER)
```

Comment:
I'm gonna bikeshed on this real fast, but I'm not going to make it a blocker.

I'd kinda like this to be a do_after, including a sound before and after the snip. This feels like something that'd be easy to mess up by mistake, especially since insulated gloves and wirecutters go hand in hand (pun intended).